### PR TITLE
Replaced hand-escaped JSON

### PR DESCRIPTION
**Pull request by Griezn**
I have replaced all hand-escaped JSON that I (and my IDE) could find. This should resolve issue #5496

### CODE REVIEW

File: src/StringUtils.cpp [hunk 1]
```diff
@@ -6,6 +6,8 @@
 #include "Globals.h"
 
 #include "fmt/printf.h"
+#include "json/json.h"
```
Comment (madmaxoft) [inline src/StringUtils.cpp:9]: The compiler complains about this line.

From what I can tell, the StringUtils file is used in other subprojects that don't have the json library added to them. You need to add the jsoncpp dependency to GrownBiomeGenVisualiser.

Details: https://builds.cuberite.org/blue/organizations/jenkins/cuberite/detail/PR-5508/1/pipeline

**Review by madmaxoft — CHANGES_REQUESTED** [review]

**Comment by madmaxoft** [comment]
The JsonCpp library is already included in Cuberite, you should NOT use `find_package` for it. Instead, use `jsoncpp_static` as a dependency for the `GrownBiomeVisualiser` target.

Ref.: https://github.com/cuberite/cuberite/blob/26b7e5ad9edebfce0cbb51a5ba606bb0b334ad2d/CMake/AddDependencies.cmake#L61-L74

**Comment by Griezn** [comment]
Whatever I do the build keeps failing. It keeps saying that it can't find `json/json.h` but I added this line to the CmakeLists.txt:
``include_directories(${PROJECT_SOURCE_DIR}/lib/jsoncpp/include)
``
And I linked the library:
`target_link_libraries(GrownBiomeGenVisualiser fmt::fmt Threads::Threads jsoncpp_static)`

But I keep getting this error:
```
/mnt/c/Users/seppe/CLionProjects/cuberite/src/StringUtils.cpp:9:10: fatal error: json/json.h: No such file or directory
    9 | #include "json/json.h"
      |          ^~~~~~~~~~~~~
compilation terminated.
make[2]: *** [Tools/MCADefrag/CMakeFiles/MCADefrag.dir/build.make:104: Tools/MCADefrag/CMakeFiles/MCADefrag.dir/__/__/src/StringUtils.cpp.o] Error 1
```

**Comment by madmaxoft** [comment]
GrownBiomeGenVisualiser now seems to work, MCADefrag is yet another tool that needs the json library added.

**Comment by madmaxoft** [comment]
Huh, we've had a JsonUtils file all along? That would have been a much better place to put the `SerializeSingleValueJsonObject` function (and probably leave out the `Json` part of the name, since it's in the namespace anyway). Sorry about my first recommendation. Seems like I've lost touch with the codebase.

If you move the function to JsonUtils, you will no longer have to add the jsoncpp dependency to all the tools. That's a big win.

File: src/StringUtils.cpp [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 #include "fmt/printf.h"
 
+
```
Comment (madmaxoft) [inline src/StringUtils.cpp:10]: Remove this change, and we're good :)

**Review by madmaxoft — CHANGES_REQUESTED** [review]

**Review by bearbin — CHANGES_REQUESTED** [review]
Please could you add yourself to the CONTRIBUTORS file to indicate your acceptance of the license, and so your efforts are recognised :)

Otherwise, this looks like a good change!

**Review by bearbin — APPROVED** [review]

**Comment by madmaxoft** [comment]
Awesome, thanks.

**Pull request merged**